package-lock.json

```diff
@@ -13669,6 +14119,44 @@
         "vfile": "^5.3.7"
       }
     },
+    "packages/myst-execute": {
+      "version": "0.0.0",
+      "license": "MIT",
+      "dependencies": {
+        "@jupyterlab/services": "^7.0.0",
+        "myst-common": "^1.1.21",
+        "unist-util-select": "^4.0.3",
+        "vfile": "^5.3.7",
+        "which": "^4.0.0"
+      },
+      "devDependencies": {
+        "@jupyterlab/nbformat": "^3.5.2",
+        "js-yaml": "^4.1.0",
+        "node-fetch": "^3.3.0"
+      }
+    },
+    "packages/myst-execute/node_modules/isexe": {
+      "version": "3.1.1",
+      "resolved": "https://registry.npmjs.org/isexe/-/isexe-3.1.1.tgz",
+      "integrity": "sha512-LpB/54B+/2J5hqQ7imZHfdU31OlgQqx7ZicVlkm9kzg9/w8GKLEcFfJl/t7DCEDueOyBAD6zCCwTO6Fzs0NoEQ==",
+      "engines": {
+        "node": ">=16"
+      }
+    },
+    "packages/myst-execute/node_modules/which": {
```

Comment:
Is this just because we didn't update the json last time?

Reply:
Yes - this one should have had the update since `package.json` changed here: https://github.com/executablebooks/mystmd/pull/866

Unrelated to this PR 🙁